tests/baselines/reference/api/typescript.d.ts

```diff
@@ -2775,18 +2775,21 @@ declare namespace ts {
         FixedPollingInterval = 0,
         PriorityPollingInterval = 1,
         DynamicPriorityPolling = 2,
-        UseFsEvents = 3,
-        UseFsEventsOnParentDirectory = 4
+        FixedChunkSizePolling = 3,
+        UseFsEvents = 4,
```

Comment:
Changing the numeric value of `UseFsEvents` won't break anyone?

Reply:
Shouldnt because you are not suppose to use "number", you are suppose to use text in tsconfig file or command line option or in protocol. This is just part of WatchOptions returned by parsing commandline